cpg_workflows/stages/gatk_sv/gatk_sv_multisample_1.py

```diff
@@ -182,13 +182,16 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
             AR_GUID_NAME: try_get_ar_guid(),
         }
 
+        # this step runs for approximately 15 hours
         jobs = add_gatk_sv_jobs(
             batch=get_batch(),
             dataset=cohort.analysis_dataset,
             wfl_name=self.name,
             input_dict=input_dict,
             expected_out_dict=expected_d,
             labels=billing_labels,
+            cromwell_status_min_poll_interval=500,
```

Comment:
Don't want to make these configurable, or at least overridable?

Reply:
That might be better... I've tried to set some reasonable intervals based on the job durations instead of loading up the config with overrides for each job.

I'll make a ticket to T-shirt size the job runtimes, then maybe have a min/max range for `small`, `medium` and `large` jobs

Reply:
You could probably roughly estimate based on how many SGs / how big the fastqs are

Reply:
From poking it in a few ways, the runtime is mostly determined by the number of tasks/sub-workflows in each stage's WDL, and the resource requirements are mainly determined by number of SGs. Both factors will have some sway